src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -365,5 +366,94 @@ public void SarifTransformerTests_ToCurrent_OneRunWitRules()
             rule.MessageStrings.Should().BeNull();
             rule.Properties.Should().BeNull();
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_OneRunWithBasicInvocation()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""invocation"": {
+                        ""commandLine"": ""CodeScanner @collections.rsp"",
+                        ""responseFiles"": {
+                          ""collections.rsp"": ""-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999""
+                        },
+                        ""startTime"": ""2016-07-16T14:18:25Z"",
+                        ""endTime"": ""2016-07-16T14:19:01Z"",
+                        ""machine"": ""BLD01"",
+                        ""account"": ""buildAgent"",
+                        ""processId"": 1218,
+                        ""fileName"": ""/bin/tools/CodeScanner"",
+                        ""workingDirectory"": ""/home/buildAgent/src"",
+                        ""environmentVariables"": {
+                          ""PATH"": ""/usr/local/bin:/bin:/bin/tools:/home/buildAgent/bin"",
+                          ""HOME"": ""/home/buildAgent"",
+                          ""TZ"": ""EST""
+                        }
+                      },
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""results"": [
+                      ]
+                    }
+                  ]
+                }";
+
+            SarifLog v2Log = TransformVersionOneToCurrent(v1LogText);
+
+            v2Log.Runs.Should().NotBeNull();
+            v2Log.Runs.Count.Should().Be(1);
+
+            Run run = v2Log.Runs[0];
+
+            run.Invocations.Should().NotBeNull();
+            run.Invocations.Count.Should().Be(1);
+
+            Invocation invocation = run.Invocations[0];
+
+            invocation.CommandLine.Should().Be("CodeScanner @collections.rsp");
+            invocation.Arguments.Should().NotBeNull();
```

Comment:
@EasyRhin0 @michaelcfanning I don't think our converter should populate `invocation.arguments` for now. The reason `arguments` exists _in addition to_ `commandLine` is so consumers don't have to parse the command line, which requires knowing and implementing the quoting and escaping rules on the SARIF producer's platform. It's alot more complicated than just breaking on white space. Unless the converter
1. knows the platform on which the producer ran, and
2. implements a command line parser for that platform

it can't reliably populate `arguments`.
SARIF doesn't define an "originatingPlatform" property, but a converter could get that information out of band, for example on its command line. So in future our converter could be enhanced with parsing for popular platforms and shells. But IMO we shouldn't take that on now.

Reply:
Done